.github/workflows/drivers.yml

```diff
@@ -73,8 +73,8 @@ jobs:
   be-tests-druid-ee:
     needs: files-changed
     if: github.event.pull_request.draft == false && needs.files-changed.outputs.backend_all == 'true'
-    runs-on: buildjet-2vcpu-ubuntu-2204
-    timeout-minutes: 60
+    runs-on: ubuntu-22.04
```

Comment:
There's not even a huge difference between these runners.
Both have 2vcpu, but BuildJet supposedly provides 8GB RAM compared to GitHub's 7GB.